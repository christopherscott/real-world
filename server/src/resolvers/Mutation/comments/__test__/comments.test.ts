// import { db, client } from '../../../../testConfig';
// import film from './fixtures/film';
// import user from './fixtures/user';

// afterEach(async () => {
//   await db.clean();
// });

// test('should add comment', async () => {
//   const { id: filmId } = await db.mutation.createFilm({ data: film }, '{ id }');

//   const { id: userId } = await db.mutation.createUser({ data: user }, '{ id }');

//   const query = `
//     mutation {
//       addComment(input: {
//         filmId: "${filmId}",
//         userId: "${userId}",
//         text: "Foo Baz"
//       }) {
//         text
//       }
//     }
//   `;

//   const responseData = await client.request(query);

//   expect(responseData).toMatchSnapshot();
// });
