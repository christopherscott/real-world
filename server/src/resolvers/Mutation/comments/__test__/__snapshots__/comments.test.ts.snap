// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return films and comment matched by search word 1`] = `
Object {
  "text": "Foo",
}
`;
